.github/workflows/integrate-and-deploy.yml

```diff
@@ -37,14 +37,11 @@ jobs:
         working-directory: docker
 
     steps:
-      -
-        uses: actions/checkout@v2
-      -
-        run: |
+      - uses: actions/checkout@v2
```

Comment:
I really like the whitespacing that was there before... mind if we either skip `yml` or add a rule to leave that whitespacing alone?

Reply:
You're saying you like having each array item start with `-` on its own line?

I think skipping `yml` files would defeat the purpose and I don't *think* prettier is configurable enough to preserve this otherwise.

Reply:
I do like them because I'm reading these contextually as "steps", not necessarily array elements... I suppose the `-` can be on the same line and I can just add lines above and below to distinguish steps clearly.